mobile_back_tflite/cpp/backend_tflite/tflite_c.cc

```diff
@@ -260,7 +260,12 @@ mlperf_backend_ptr_t mlperf_backend_create(
     } else if (strcmp(configs->accelerator, "neuron") == 0) {
       // The kOptimizationBatchProcessor doesn't work yet.
       // Use NNAPI instead.
-      if (configs->batch_size != 1) {
+      //
+      // Here we use the batch size of the task to check if this
+      // is for offline/batch scenario, if batch size is not
+      // larger than 1, assume it's offline and use NNAPI delegate
+      // to handle it.
```

Comment:
It doesn't really help to understand **_why_** we use `configs->batch_size` instead of `backend_data->real_batch_size`.
Could you rephrase the comment to make it more clear?